blivet/devicefactory.py

```diff
@@ -1694,8 +1721,8 @@ class BTRFSFactory(DeviceFactory):
     child_factory_class = PartitionSetFactory
     child_factory_fstype = "btrfs"
 
-    def __init__(self, storage, size, disks, **kwargs):
-        super(BTRFSFactory, self).__init__(storage, size, disks, **kwargs)
+    def __init__(self, *args, **kwargs):
```

Comment:
I think ``*args`` should be dropped here and in all the above ``__init__`` functions.